ddl/ddl.go

```diff
@@ -1507,10 +1507,10 @@ func resumePausedJob(se *sess.Session, job *model.Job,
 func processJobs(process func(*sess.Session, *model.Job, model.AdminCommandOperator) (err error),
 	sessCtx sessionctx.Context,
 	ids []int64,
-	byWho model.AdminCommandOperator) ([]error, error) {
+	byWho model.AdminCommandOperator) (jobErrs []error, err error) {
 	failpoint.Inject("mockFailedCommandOnConcurencyDDL", func(val failpoint.Value) {
 		if val.(bool) {
-			failpoint.Return(nil, errors.New("mock commit error"))
+			failpoint.Return(nil, errors.New("mock failed admin command on ddl jobs"))
```

Comment:
Do we need to update this error to distinguish error messages with https://github.com/pingcap/tidb/pull/44007/files#diff-a185df836b8926c4afe351d4def1b56d1e282c65aabe6677c2fa2c54590cc4b2R1568？